hiv_synthesis.sas

```diff
@@ -17003,7 +17002,7 @@ if 15 <= age      and (death = . or caldate&j = death ) then do;
 
 	s_all_prep_criteria + all_prep_criteria ; s_all_prep_criteria_hivneg + all_prep_criteria_hivneg; s_prep_elig_hivneg + prep_elig_hivneg ;
 	s_prep_elig_hivneg_onprep + prep_elig_hivneg_onprep ;  s_prep_elig_onprep + prep_elig_onprep ;s_prep_elig_onprep_inj + prep_elig_onprep_inj;
-	s_started_prep_inj_hiv + started_prep_inj_hiv; s_prep_elig_onprep_vr + prep_elig_onprep_vr; 	s_started_prep_vr_hiv + started_prep_vr_hiv;
+	s_started_prep_oral_hiv + started_prep_oral_hiv; s_started_prep_inj_hiv + started_prep_inj_hiv; s_prep_elig_onprep_vr + prep_elig_onprep_vr; 	s_started_prep_vr_hiv + started_prep_vr_hiv;
```

Comment:
just noticed that here you added:
- s_started_prep_oral_hiv
- s_prep_elig_onprep_vr
Is it intentional that you added the latter?

Reply:
I think that s_prep_elig_onprep_vr was already there but the order is a bit confusing and I can also see that s_prep_elig_onprep_oral doesn't exist. I will add that in and reorder the outputs so it's a bit clearer. Now inputted is:
```
s_prep_elig_onprep s_prep_elig_onprep_oral s_prep_elig_onprep_inj s_prep_elig_onprep_vr 
s_started_prep_any_hiv s_started_prep_oral_hiv s_started_prep_inj_hiv s_started_prep_vr_hiv
```